packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -74,27 +75,21 @@ function handler(app: Application) {
       return;
     }
 
-    // update each settings section
-    try {
-      await handler.updateRepoOptions(r, context);
-    } catch (err) {
-      console.log(`Error updating repo options for ${repo}`);
-      console.log(err);
-    }
-
-    try {
-      await handler.updateMasterBranchProtection(r, context);
-    } catch (err) {
-      console.log(`Error updating master branch protection for ${repo}`);
-      console.log(err);
+    if (context.payload.org !== owner) {
+      console.log(`skipping run for ${context.payload.org}`);
+      return;
     }
 
-    try {
-      await handler.updateRepoTeams(r, context);
-    } catch (err) {
-      console.log(`Error updating repo in org for ${repo}`);
-      console.log(err);
-    }
+    const start = new Date().getTime();
+    // update each settings section
+    await Promise.all([
+      await handler.updateRepoOptions(r, context),
```

Comment:
when you do `Promise.all` you don't need the await, this would actually still be sequential.